manim/animation/animation.py

```diff
@@ -144,6 +144,7 @@ def __init__(
         **kwargs,
     ) -> None:
         self._typecheck_input(mobject)
+        self._run_time: float = run_time
         self.run_time: float = run_time
```

Comment:
Do we really need to set the actual attribute and the property here?

Reply:
Not really. I just removed that line.